src/inc/quic_platform_posix.h

```diff
@@ -1066,35 +1201,48 @@ CxPlatSqeInitialize(
     )
 {
     UNREFERENCED_PARAMETER(queue);
-    sqe->Completion = completion;
+    sqe->ClassicCompletion = completion;
+    sqe->Completion = CxPlatSqeClassicCompletion;
+#if DEBUG
+    sqe->Signature = CXPLAT_SQE_SIGNATURE_INITIALIZED;
+#endif
     return TRUE;
 }
 
 QUIC_INLINE
-void
-CxPlatSqeCleanup(
+BOOLEAN
+CxPlatSqeInitialize2(
```

Comment:
Initialize**2**?!

Reply:
Yeah, this deserves a better name - I'll call it a batched SQE